packages/invoices/src/OutboundShipment/DetailView/tabs/GeneralTab.tsx

```diff
@@ -18,12 +17,7 @@ export const GeneralTabComponent: FC<GeneralTabProps<ItemRow>> = ({
   data,
   columns,
 }) => {
-  const numberOfRows = useRowRenderCount();
-  const { pagination } = usePagination(numberOfRows);
-
-  useEffect(() => {
-    pagination.onChangeFirst(numberOfRows);
-  }, [numberOfRows, pagination.first]);
+  const { pagination } = usePagination(20);
```

Comment:
I wonder if adding a default to the `usePagination` would be better and avoid having the `20` in a number of different places